### PR TITLE
feat: chinese page in go-api/basic/

### PR DESCRIPTION
**Pull request by chivalryq**

### CODE REVIEW

File: content/go-api/basics/evaluating.zh.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+记得吗，`number` 可以表示整数和小数。
+这意味着这两个限制在格上是平等的。
+可以设置为 `i: int & >10`，这个限制就可以被 `int` 容纳了。
+
+_另外请注意，如果所有 Value 都在同一个文件里，意味着已经聚合过了_
+
+
+### Value.Eval()
+
+评估（Eval）函数会解析所有引用，保证具体性，发现所有错误，并最终产生一个新的值。
```
Comment (h1z3y3) [inline content/go-api/basics/evaluating.zh.md:82]: Could `Eval` be `求值`?

File: content/go-api/basics/inspecting.zh.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+[ReferencePath](https://pkg.go.dev/cuelang.org/go@v0.4.0/cue#Value.ReferencePath),
+[Dereference](https://pkg.go.dev/cuelang.org/go@v0.4.0/cue#Dereference)
+
+### Exists 和 IsConcrete
+
+当你查找一个 Value 时，你怎么知道你收否找到了呢？这就用到了 `Exists` 方法。
+
+`IsConcrete` 方法可以表明一个原子字段有数据或是一个错误。
+对于列表或结构体类型，如果他们存在则返回true。
+当使用析取和默认值时...
```
Comment (h1z3y3) [inline content/go-api/basics/inspecting.zh.md:58]: `disjunctions` could be `分隔符`, according [this page](https://cuetorials.com/zh/overview/foundations/#%E5%88%86%E7%A6%BB%E7%AC%A6)

Please correct me in your PR, I translated it as `分离符` when I hadn't truly understood it.

File: content/go-api/basics/inspecting.zh.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+`IsConcrete` 方法可以表明一个原子字段有数据或是一个错误。
+对于列表或结构体类型，如果他们存在则返回true。
+当使用析取和默认值时... 
+
+{{< codePane2
+	file1="code/go-api/basics/inspecting/exists.go"  lang1="go"  title1="exists.go"
+	file2="code/go-api/basics/inspecting/exists.txt" lang2="txt" title2="go run exists.go"
+>}}
+
+使用 `Validate` 方法检查一整个value是否时具体的。
```
Comment (h1z3y3) [inline content/go-api/basics/inspecting.zh.md:65]: `是`

File: content/go-api/basics/printing.zh.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+### 使用 go/fmt 打印
+
+{{< codePane2 
+	file1="code/go-api/basics/printing/print.go" title1="print.go" lang1="go"
+	file2="code/go-api/basics/printing/print.html"  title2="go run print.go"
+>}}
+
+### 使用 cue/format 打印
+
+当运行 `cue eval` 和 `cue def`的时候 我们可以通过 flag 来控制打印格式选项。
```
Comment (h1z3y3) [inline content/go-api/basics/printing.zh.md:29]: 你这里有些不统一，有的代码前面有空格，有的没有。

我这边是习惯所有的英文字符前后都加空格。

More information: https://github.com/DavidAnson/markdownlint

File: content/go-api/basics/traversing.zh.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+- iterate
+- iteration
+- custom
+- walk
+weight: 70
+---
+
+{{<lead>}}
+有几种方式遍历 Value。例如基于路径的查找，迭代、以及遍历语法树。
+其中一些时可配置的，与不同的字段标签有关，例如常规的、隐藏的、可选的，以及定义。
```
Comment (h1z3y3) [inline content/go-api/basics/traversing.zh.md:17]: `是`

File: content/go-api/basics/evaluating.zh.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+- validate
+- unify
+- subsume
+- eval
+- options
+weight: 50
+---
+
+{{<lead>}}
+评估是 CUE 的核心能力，并且是其他一切功能（如统一和验证）的底层能力。
```
[the remark this answers is not in the document]
Comment (chivalryq) [reply content/go-api/basics/evaluating.zh.md:16]: Good advice!

File: content/go-api/basics/context.zh.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+{{< codePane2
+	file1="code/go-api/basics/context/context.go" lang1="go" title1="context.go"
+	file2="code/go-api/basics/context/context.html"  title2="go run context.go"
+>}}
+
+
+### 编译时指定范围（Scope）
+
+如果值（value）和类型（schema）在不同的字符串中，
```
[the remark this answers is not in the document]
Comment (chivalryq) [reply content/go-api/basics/context.zh.md:38]: It's like a complicated type so I translate it to `结构`. I'm not sure about that however.

File: content/go-api/basics/inspecting.zh.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+	file2="code/go-api/basics/inspecting/reference.txt" lang2="txt" title2="go run reference.go"
+>}}
+
+文档:
+[ReferencePath](https://pkg.go.dev/cuelang.org/go@v0.4.0/cue#Value.ReferencePath),
+[Dereference](https://pkg.go.dev/cuelang.org/go@v0.4.0/cue#Dereference)
+
+### Exists 和 IsConcrete
+
+当你查找一个 Value 时，你怎么知道你收否找到了呢？这就用到了 `Exists` 方法。
```
[the remark this answers is not in the document]
Comment (chivalryq) [reply content/go-api/basics/inspecting.zh.md:54]: yeah, that's better!

File: content/go-api/basics/inspecting.zh.md (hunk 2 above)
Comment (chivalryq) [reply content/go-api/basics/inspecting.zh.md:58]: `析取` is more appropriate IMO. It's like "union" or `并集`。

File: content/go-api/basics/printing.zh.md (hunk 4 above)
Comment (chivalryq) [reply content/go-api/basics/printing.zh.md:29]: Yes, I tried.... But I missed someplace. How do you ensure all English words and code are gapped? Any human-friendly methods or tools?

**Comment by chivalryq** [comment]
Nice Catch! @h1z3y3 I have resolved most of them. Could you check again?

File: content/go-api/basics/printing.zh.md (hunk 4 above)
Comment (h1z3y3) [reply content/go-api/basics/printing.zh.md]: Actually it's not required by https://github.com/DavidAnson/markdownlint. 

You can download its command CLI to lint your markdown codes, it also provides github bot in Github Action.

But I don't use it in this website. 😂

File: content/go-api/basics/context.zh.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+你可以使用 `cue.Scope` 选项来给 `cue.Context` 对象提供一个更具体的上下文。
+
+{{< codePane2
+	file1="code/go-api/basics/context/scope.go" lang1="go" title1="scope.go"
+	file2="code/go-api/basics/context/scope.html"  title2="go run scope.go"
+>}}
+
+### Encoding Values from Go
```
[the remark this answers is not in the document]
Comment (h1z3y3) [reply content/go-api/basics/context.zh.md:46]: I think this line should be deleted.

File: content/go-api/basics/context.zh.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (h1z3y3) [reply content/go-api/basics/context.zh.md:38]: Are you sure ? You're using `类型` 😂

**Comment by h1z3y3** [comment]
@verdverm  It's ready to merge.

**Comment by verdverm** [comment]
Thanks @chivalryq & @h1z3y3

File: content/go-api/basics/context.zh.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (chivalryq) [reply content/go-api/basics/context.zh.md:38]: Sorry, I don't know why I type that. I mean `类型`...

**Pull request merged**